mne/evoked.py

```diff
@@ -377,7 +377,7 @@ def plot(self, picks=None, exclude='bads', unit=True, show=True, ylim=None,
         show : bool
             Call pyplot.show() at the end or not.
         ylim : dict
-            ylim for plots. e.g. ylim = dict(eeg=[-200e-6, 200e-6])
+            ylim for plots. e.g. ylim = dict(eeg=[-200, 200])
```

Comment:
doesn't seem to work for me. Not sure if it needs a fix in the docstring or the logic